src/_gettsim/transfers/rente.py

```diff
@@ -19,8 +19,181 @@ def sum_ges_rente_priv_rente_m(priv_rente_m: float, ges_rente_m: float) -> float
     return out
 
 
-@policy_info(params_key_for_rounding="ges_rente")
-def ges_rente_vor_grundr_m(
+@policy_info(
+    start_date="2021-01-01",
+    params_key_for_rounding="ges_rente",
+    name_in_dag="ges_rente_m",
+)
+def ges_rente_mit_grundrente_m(
+    bruttorente_m: float,
+    grundr_zuschlag_m: float,
+    rentner: bool,
+) -> float:
+    """Calculate total individual public pension including Grundrentenzuschlag. Is only
```

Comment:
```suggestion
    """Calculate total individual public pension including Grundrentenzuschlag.
```
No need to re-state what is obvious from the decorator.